examples/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine
```

Comment:
For the consistency could you please:
* use form  `scratch`? 
* `COPY collector-linux /go/bin/`

Reply:
Dockerfile should be in `examples/hotrod` directory

Reply:
@pavolloffay is this collector-linux folder in jaeger project? I didn't find it